packages/pectin-core/test/pectin-core.test.js

```diff
@@ -175,7 +176,10 @@ describe('pectin-core', () => {
                 },
             }),
             src: Dir({
-                'index.js': File('export default function foo() {};'),
+                'test.svg': File(`test`),
+                'index.js': File(
+                    `import svgTest from './test.svg'; export default function foo() {};`
```

Comment:
I think this needs to use the `svgTest` import, otherwise it gets stripped away. Like, return it from `foo()` or something. The snapshot should be changing, basically.